tvb_build/docker/Dockerfile-build

```diff
@@ -4,8 +4,7 @@ ARG TVB_USER=tvb_user
 ARG USER_HOME=/home/$TVB_USER
 ARG TVB_STORAGE=$USER_HOME/TVB_STORAGE
 ARG KEYCLOAK_CONFIG=$USER_HOME/keycloak_config
-ARG JUPYTER_CONFIG=$USER_HOME/.jupyter
-
+ARG JUPYTER_CONFIG=$USER_HOME/.jupyter>
```

Comment:
This this change ?

Reply:
I will revert this change to the original one.